scripts/build.sh

```diff
@@ -146,7 +146,7 @@ if [[ "$xcode_major" -lt 15 ]]; then
 else
   ios_flags=(
     -sdk 'iphonesimulator'
-    -destination 'platform=iOS Simulator,name=iPhone 15'
+    -destination 'platform=iOS Simulator,name=iPhone 16,OS=18.0'
```

Comment:
Not sure the OS should be specified here

Reply:
If not it seems to default to 18.1 (beta), but it's also more brittle to specify it. Unsure what to do.

Reply:
Removed the `OS=18.0` and updated to use the iPhone 15 on Xcode 15 (and iPhone 16 on Xcode 16).